compiler/rustc_sanitizers/src/cfi/typeid/itanium_cxx_abi/encode.rs

```diff
@@ -1,6 +1,3 @@
-/// Type metadata identifiers (using Itanium C++ ABI mangling for encoding) for LLVM Control Flow
-/// Integrity (CFI) and cross-language LLVM CFI support.
-///
 /// Encodes type metadata identifiers for LLVM CFI and cross-language LLVM CFI support using Itanium
```

Comment:
I can't select all of the lines because GitHub thinks this is a rename, but same doc comment change as the others.

Reply:
Done.